web/packages/extension/src/background.ts

```diff
@@ -9,6 +9,75 @@ async function contentScriptRegistered() {
 }
 
 async function enable() {
+    if (utils.declarativeNetRequest) {
```

Comment:
This needs feature detection, which I asked about here: https://github.com/w3c/webextensions/issues/638

Reply:
Even if that issue is addressed, it won't help as it won't impact older versions of Chrome. My plan is to do something like the following:
```ts
if (utils.declarativeNetRequest && typeof browser !== "undefined") {
```
This will work if https://issues.chromium.org/issues/40556351 gets fixed after https://issues.chromium.org/issues/40727004, once both have been fixed.

Reply:
Currently doing this in the way suggested here: https://github.com/w3c/webextensions/issues/460#issuecomment-2169017147

> Perhaps not an ideal solution but checking the user agent of the browser for a Chrome version may help detect if response header conditions are supported

Reply:
Changed to the suggestion on https://github.com/w3c/webextensions/issues/638#issuecomment-2181124486